test/isUrlRequest.test.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+const assert = require("assert");
+const loaderUtils = require("../");
+
+describe("isUrlRequest()", () => {
+	[
+		["test url", "about:blank", undefined, false],
+	].forEach(test => {
+		it("should isUrlRequest " + test[0] + " " + test[1] + " " + test[2], () => {
```

Comment:
You can run the tests with `npm test`. You'll see that the output makes more sense this way.